.circleci/config.yml

```diff
@@ -164,6 +164,12 @@ orbs:
         <<: *test_job_default
         steps:
           - checkout
+          - run:
+              name: Know the environment
+              command: |
+                ruby -v
+                gem -v
+                bundle -v
```

Comment:
Should this section be kept?

Reply:
I like to keep those debug info handy, and they are cheap to keep